pycon/settings.py

```diff
@@ -63,13 +64,11 @@ def _(x):
 # DATABASE
 # --------
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': SITE_DATA_ROOT + '/epcon.db',
-    }
+    'default': dj_database_url.config(
+        default='sqlite:///{}/epcon.db'.format(SITE_DATA_ROOT)
+    ),
```

Comment:
we could just use DATABASE_PATH as env variable (with decouple, like other variables) here, because we don't change the protocol.

Reply:
Well, using `dj_database_url`, we can set `DATABASE_URL` and override just the db path or the protocol and db path; if we want to move to postgres at one point, I think it's a change we'll need at some point anyway.